examples/experimental/bitmap/app.js

```diff
@@ -0,0 +1,132 @@
+/* global window,document */
+import React, {Component} from 'react';
+import {render} from 'react-dom';
+import DeckGL, {COORDINATE_SYSTEM, PerspectiveView, experimental} from 'deck.gl';
+import {BitmapLayer} from '@deck.gl/experimental-layers';
+
+const {OrbitController} = experimental;
+
+const IMAGES = {
+  NEGX: 'images/skynegx1.png',
+  NEGY: 'images/skynegy1.png',
+  NEGZ: 'images/skynegz1.png',
+  POSX: 'images/skyposx1.png',
+  POSY: 'images/skyposy1.png',
+  POSZ: 'images/skyposz1.png'
+};
+
+function toRadian(degree) {
+  return Math.PI * degree / 180;
+}
+
+function toRadians(degrees) {
+  return degrees.map(d => toRadian(d));
+}
+
+const defaultViewState = {
+  eye: [0, 0, 0],
+  lookAt: [0, 0, 0],
+  up: [0, 1, 0],
+  zoom: 0.1,
+  width: 100,
+  height: 100,
+  distance: 1
+};
+
+class Root extends Component {
+  constructor(props) {
+    super(props);
+    this._onViewStateChange = this._onViewStateChange.bind(this);
+    this._resize = this._resize.bind(this);
+    this.state = {viewState: defaultViewState};
+  }
+
+  componentDidMount() {
```

Comment:
Use `<DeckGL width='100%' height='100%' instead of manually tracking size/resize?